src/tribler.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/sh
 # Run Tribler from source tree
 
+# Options
+HEADLESS=false
+# Use a custom env
+#PYTHONPATH=/home/user/Downloads/tribler_env/lib/python3.11/site-packages
```

Comment:
These lines appear to be specific to a particular user and it doesn't seem appropriate to commit them to the main repository.
```suggestion

```